neuvector-controller-5.3.yaml

```diff
@@ -38,9 +39,40 @@ pipeline:
       packages: .
       output: controller
 
+  - runs: |
+      # Create Folders
+      mkdir -p ${{targets.contextdir}}/etc
+      mkdir -p ${{targets.contextdir}}/usr/local/bin
+      mkdir -p ${{targets.contextdir}}/etc/neuvector/templates
+      mkdir -p ${{targets.contextdir}}/etc/tmp
+
+      # Scripts
+      cp scripts/sysctl.conf ${{targets.contextdir}}/etc/
+      cp scripts/teardown.sh ${{targets.contextdir}}/usr/local/bin/
+      cp scripts/runtime-gdb.py ${{targets.contextdir}}/usr/local/bin/
+
+      # Template
+      cp templates/podTemplate.json ${{targets.contextdir}}/etc/neuvector/templates/podTemplate.json
+      wget https://raw.githubusercontent.com/neuvector/manifests/main/build/share/.nvcontainer -P ${{targets.contextdir}}/usr/local/bin/.nvcontainer
+
+      # symlink
+      ln -sf /usr/bin/controller ${{targets.contextdir}}/usr/local/bin/controller
+      ln -sf /usr/bin/opa ${{targets.contextdir}}/usr/local/bin/opa
+      ln -sf /bin/consul ${{targets.contextdir}}/usr/local/bin/consul
+
+      # Certs
+      mkdir -p ${{targets.contextdir}}/etc/neuvector/certs
+      wget https://raw.githubusercontent.com/neuvector/manifests/main/build/share/etc/neuvector/certs/ssl-cert.key -P ${{targets.contextdir}}/etc/neuvector/certs/
+      wget https://raw.githubusercontent.com/neuvector/manifests/main/build/share/etc/neuvector/certs/ssl-cert.pem -P ${{targets.contextdir}}/etc/neuvector/certs/
```

Comment:
I'd like these to be openssl generated instead of downloaded, even if they're just placeholders. Does the controller 
 need them to exist to run, is there anything specific to the key/cert that requires using their files?